synapse/handlers/message.py

```diff
@@ -1293,13 +1295,12 @@ async def _validate_event_relation(self, event: EventBase) -> None:
     async def handle_new_client_event(
         self,
         requester: Requester,
-        event: EventBase,
-        context: EventContext,
+        events_and_context: List[Tuple[EventBase, EventContext]],
         ratelimit: bool = True,
         extra_users: Optional[List[UserID]] = None,
         ignore_shadow_ban: bool = False,
     ) -> EventBase:
-        """Processes a new event.
```

Comment:
Can you add a note to these docstrings that an error handling any one of these events will cause us to drop all the events